cmd/podman/images/search.go

```diff
@@ -180,3 +179,39 @@ func imageSearch(cmd *cobra.Command, args []string) error {
 
 	return tmpl.Execute(w, searchReport)
 }
+
+func printJson(v interface{}) error {
+	prettyJSON, err := json.MarshalIndent(v, "", "    ")
+	if err != nil {
+		return err
+	}
+	fmt.Println(string(prettyJSON))
+	return nil
+}
+
+func buildListTagsJson(searchReport []entities.ImageSearchReport) interface{} {
+	entries := []struct {
+		Name string
+		Tags []string
+	}{}
```

Comment:
Make a type definition for the struct outside of the function. Like this:
```golang
type listTagEntry struct {
    Name string
    Tags []string
}
```
Use this type for `entries`, `newElem` and also as return value for the function. `interface{}` should be avoided.

Reply:
Ok, thank you for your advices.